### DISCUSSION TITLE
How can I verify that hcxdumptool is not sending DEAUTHENTICATION frames

### DISCUSSION THREAD

**Discussion by ZerBea** [ZerBea/hcxdumptool#395] [Q&A]
To verify that hcxdumptool is really not sending DEAUTHENTICATION frames on option "--disable_deauthentication" run Wireshark in parallel on the same(!) interface.
Apply filter to monitor outgoing traffic and DEAUTHENTICATION frames:
`(wlan.fc.type_subtype==0x0c) && (radiotap.present.dbm_antsignal == False)`

**Comment by ghost** [comment]


**Reply by ZerBea** [reply]
You can always run tshark or Wireshark on the same interface as hcxdumptool.
You can always use the same capture filter (BPF) syntax on hcxdumptool and tshark or Wireshark.

An example:
target AP MAC is 11:22:33:44:55:66
Terminal 1:
```
$ hcxdumptool --bpfc="wlan addr3 11:22:33:44:55:66" > attack.bpf
$ sudo hcxdumptool -i INTERFACE --rds=1 -F --bpf=attack.bpf -w hcxdump.pcapng
```

Terminal 2:
`$ tshark -i INTERFACE -f "wlan addr3 11:22:33:44:55:66"`

Doing this, you can easy improve your BPF.

Do you see the benefits of BPF compared to stupid command line options?
